src/sql/workbench/services/query/common/queryRunner.ts

```diff
@@ -281,7 +282,8 @@ export default class QueryRunner extends Disposable {
 			message: batch.range ? nls.localize('query.message.startQueryWithRange', "Started executing query at Line {0}", batch.range.startLineNumber) : nls.localize('query.message.startQuery', "Started executing batch {0}", batch.id),
 			time: batch.executionStart,
 			range: batch.range,
-			isError: false
+			isError: false,
+			isQueryStart: true
```

Comment:
I'd consider having this be something like `messageType` (with the value being an enum of types such as `normal` and `queryStart`) instead so that if we have other types of messages we want to filter or do something special with we can easily add those without bloating the object.